packages/keystone/tasks.js

```diff
@@ -63,9 +100,19 @@ function removeCronTask (name, cron, opts = {}) {
     REMOVE_CRON_TASKS.push([name, taskOpts])
 }
 
-async function _scheduleRemoteTask (name, preparedArgs, preparedOpts) {
-    logger.info({ msg: 'Scheduling task', name, data: { preparedArgs, preparedOpts } })
-    const job = await taskQueue.add(name, { args: preparedArgs }, preparedOpts)
+async function _scheduleRemoteTask (name, preparedArgs, preparedOpts, queue = DEFAULT_QUEUE_NAME) {
+    logger.info({ msg: 'Scheduling task', name, queue, data: { preparedArgs, preparedOpts } })
```

Comment:
Maybe `meta` instead of `data` to avoid indexing all args later?

Reply:
I've changed `data` -> `meta`